Formula/ibex.rb

```diff
@@ -12,37 +24,82 @@ class Ibex < Formula
     sha256 "41327b6a0da9b8b2ed888c3c2c33d5f6ff061ea13eceb4b9e8b6e660049d624d" => :mountain_lion
   end
 
-  option "with-java", "Build Java bindings for Choco solver."
+  option "with-java", "Enable Java bindings for CHOCO solver."
+  option "with-affine", "Use Affine Arithmetic plugin"
```

Comment:
We probably don't want to add _this many_ options unless there proves to be significant demand for them. Options are quite maintenance intensive. Can we trim this list a bit at all?